### PR TITLE
docs: Update our list of volunteers

### PR DESCRIPTION
**Pull request by jonaharagon**
Changes proposed in this PR:

- add @redoomed1 @kimg45 @ph00lt0 
- update allcontributorsrc with missing github contributors

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -16,37 +16,19 @@ schema:
...
-
-We additionally launched our new discussion forum at [discuss.privacyguides.net](https://discuss.privacyguides.net) as a community platform to share ideas and ask questions about our mission. This augments our existing community on Matrix, and replaced our previous GitHub Discussions platform, decreasing our reliance on proprietary discussion platforms.
-
-So far in 2023 we've launched international translations of our website in [French](/fr/), [Hebrew](/he/), and [Dutch](/nl/), with more languages on the way, made possible by our excellent translation team on [Crowdin](https://crowdin.com/project/privacyguides). We plan to continue carrying forward our mission of outreach and education, and finding ways to more clearly highlight the dangers of a lack of privacy awareness in the modern digital age, and the prevalence and harms of security breaches across the technology industry.
-
-## Our Team
+## Executive Committee
 <!-- markdownlint-disable MD030 -->
 
+The project executive committee consists of five volunteers charged with management of the [MAGIC Privacy Guides Fund](https://magicgrants.org/funds/privacy_guides/), making most critical project-related decisions.
```
Comment (redoomed1) [inline docs/about.md:30]:
```suggestion
The project executive committee consists of five volunteers charged with management of the [MAGIC Privacy Guides Fund](https://magicgrants.org/funds/privacy_guides), making most critical project-related decisions.
```

File: docs/about.md [hunk 2]
```diff
@@ -56,69 +38,102 @@ So far in 2023 we've launched international translations of our website in [Fren
...
-    [:simple-mastodon: Mastodon](https://mastodon.neat.computer/@oliviablob "@oliviablob@neat.computer"){rel=me}
+    [:simple-github:](https://github.com/hook9 "GitHub")
+    [:simple-mastodon:](https://mastodon.neat.computer/@oliviablob "@oliviablob@neat.computer"){rel=me}
 
 </div>
 
-Additionally, [many people](contributors.md) have made contributions to the project. You can too, we're open source on GitHub, and accepting translation suggestions on [Crowdin](https://crowdin.com/project/privacyguides).
+## Volunteer Team
+
+A number of other contributors have volunteered their time to review and approve changes to this website, and keep the website up to date. Changes require 2+ approvals before they can be merged. In addition to the executive committee members above, volunteers [trusted](https://github.com/orgs/privacyguides/people) to review pull requests include:
```
Comment (redoomed1) [inline docs/about.md:104]: Just a small distinction (green checks in contrast to gray checks)

```suggestion
A number of other contributors have volunteered their time to review and approve changes to this website, and keep the website up to date. Changes require 2+ approvals from team members before they can be merged. In addition to the executive committee members above, volunteers [trusted](https://github.com/orgs/privacyguides/people) to review pull requests include:
```

File: docs/about.md [hunk 3]
```diff
@@ -56,69 +38,102 @@ So far in 2023 we've launched international translations of our website in [Fren
...
+
+- [@kimg45](https://github.com/kimg45)
+- [@ph00lt0](https://github.com/ph00lt0)
+- [@redoomed1](https://github.com/redoomed1)
+
+We also especially thank our dedicated moderation team on Matrix and our forum: *Austin Huang*, *namazso*, *hik*, *riley*, and *Valynor*.
+
+Additionally, [many other people](about/contributors.md) have made contributions to the project. You can too, we're open source on GitHub, and accepting translation suggestions on [Crowdin](https://crowdin.com/project/privacyguides).
```
Comment (redoomed1) [inline docs/about.md:116]:
```suggestion
Additionally, [many other people](about/contributors.md) have made contributions to the project. You can too! We're open source on GitHub, and accepting translation suggestions on [Crowdin](https://crowdin.com/project/privacyguides).
```

**Review by freddy-m — APPROVED** [review]
This looks great - thanks to all our volunteers and contributors!

File: docs/about/donate.md [hunk 4]
```diff
@@ -1,43 +1,51 @@
...
 :   Traffic to this website uses hundreds of gigabytes of data per month, we use a variety of service providers to keep up with this traffic.
 
+**Domain Registrations**
+
+:   We have a few domain names like `privacyguides.org` which cost us around $10 yearly to maintain their registration.
+
 **Online Services**
 
-:   We host [internet services](https://privacyguides.net) for testing and showcasing different privacy-products we like and [recommend](../tools.md). Some of which are made publicly available for our community's use (SearXNG, Tor, etc.), and some are provided for our team members (email, etc.).
+:   We host [internet services](services.md) for testing and showcasing different privacy-products we like and [recommend](../tools.md). Some of which are made publicly available for our community's use (SearXNG, Tor, etc.), and some are provided for our team members (email, etc.).
```
Comment (redoomed1) [inline docs/about/donate.md:31]:
```suggestion
:   We host [internet services](services.md) for testing and showcasing different privacy-products we like and [recommend](../tools.md). Some of them are made publicly available for our community's use (SearXNG, Tor, etc.), and some are provided for our team members (email, etc.).
```

File: docs/about/donate.md [hunk 5]
```diff
@@ -1,43 +1,51 @@
...
 
 **Product Purchases**
 
 :   We occasionally purchase products and services for the purposes of testing our [recommended tools](../tools.md).
 
-Your donation will go to a dedicated fund within [MAGIC Grants](https://magicgrants.org), a 501(c)(3) organization. The funds will only be used for this project specifically. You may qualify for a tax deduction. If you need a donation receipt, please email <info@magicgrants.org>.
+Your donation will go to a [dedicated fund](https://magicgrants.org/funds/privacy_guides/) within [MAGIC Grants](https://magicgrants.org/), a 501(c)(3) organization. The funds will only be used for this project specifically. You may qualify for a tax deduction. If you need a donation receipt, please email <info@magicgrants.org>.
```
Comment (redoomed1) [inline docs/about/donate.md:37]:
```suggestion
Your donation will go to a [dedicated fund](https://magicgrants.org/funds/privacy_guides) within [MAGIC Grants](https://magicgrants.org), a 501(c)(3) organization. The funds will only be used for this project specifically. You may qualify for a tax deduction. If you need a donation receipt, please email <info@magicgrants.org>.
```

**Review by redoomed1 — CHANGES_REQUESTED** [review]

**Review by blacklight447 — APPROVED** [review]
Ready to go.

File: docs/about/donate.md [hunk 6]
```diff
@@ -1,43 +1,51 @@
...
+
+## Merchandise
+
+Another option to support us is by buying our merchandise from HelloTux. We get a small commission for each item sold, and you get a quality product to show for it.
+
+[Buy on HelloTux.com](https://hellotux.com/privacyguides){ class="md-button" }
+
+## Non-Financial Support
+
+It takes a lot of [people](contributors.md) and [work](https://github.com/privacyguides/privacyguides.org/pulse/monthly) to keep Privacy Guides up to date and spreading the word about privacy and mass surveillance. If you're looking for other ways to help out, consider getting involved by [editing the site](https://github.com/privacyguides/privacyguides.org), [joining our forum](https://discuss.privacyguides.net), or [contributing translations](https://crowdin.com/project/privacyguides).
```
Comment (redoomed1) [inline docs/about/donate.md:51]:
```suggestion
It takes a lot of [people](contributors.md) and [work](https://github.com/privacyguides/privacyguides.org/pulse/monthly) to keep Privacy Guides up to date and spread the word about privacy and mass surveillance. If you're looking for other ways to help out, consider getting involved by [editing the site](https://github.com/privacyguides/privacyguides.org), [joining our forum](https://discuss.privacyguides.net), or [contributing translations](https://crowdin.com/project/privacyguides).
```

**Review by redoomed1 — APPROVED** [review]
Other than this minor suggestion, LGTM.

**Pull request merged**